spec/openapi.yaml

```diff
@@ -9316,7 +9316,7 @@ paths:
               examples:
                 response:
                   value: >-
-                    Account,Project,Job,Translation Resource,Agency,Target
+                    Account,Project,Job Uid,Job,Translation Resource,Agency,Target
```

Comment:
Seems you missed the actual job uid in the second row of the example

Reply:
fixed